app-web/src/constants/routes.js

```diff
@@ -13,7 +13,7 @@ export const MAIN_NAV_ROUTES = {
   },
   COLLECTIONS: {
     to: '/collections',
```

Comment:
does changing this to /topics break anything?

Reply:
Yes, I believe I got a 404 when I tried that.